sycl/doc/extensions/SPIRV/SPV_INTEL_joint_matrix.asciidoc

```diff
@@ -477,4 +492,5 @@ Revision History
 |Rev|Date|Author|Changes
 |1|2021-02-16|Alexey Sotkin|Initial revision
 |1|2021-09-06|Dmitry Sidorov|Split OpJointMatrixMadINTEL instruction into 4
+|1|2021-12-28|Dmitry Sidorov|Add Joint Matrix to Composite definition
```

Comment:
Why it is still "1" and why in the Revision table the revision is "B"?

Reply:
> why in the Revision table the revision is "B"?

Well, it was "A" there :)